# https://github.com/Perl/perl5/pull/24042
# MM.pm: treat undefined mtimes as 0 to avoid warnings
#
# Insert // 0 for mtime comparisons so missing pod or Makefile are treated as 0. Preserves original skip behavior while preventing "uninitialized value" warnings in numeric comparisons.
# 
# This suppresses 3077 uninitialized occurrences when building ZoneMinder on Gentoo using cmake 4.1.2 perl 5.42.0
# 
# Use of uninitialized value in numeric gt (>) at /usr/lib64/perl5/5.42/ExtUtils/Command/MM.pm line 147.
#
diff --git a/cpan/ExtUtils-MakeMaker/lib/ExtUtils/Command/MM.pm b/cpan/ExtUtils-MakeMaker/lib/ExtUtils/Command/MM.pm
index 89f4c16ded..d8a0b3b426 100644
--- a/cpan/ExtUtils-MakeMaker/lib/ExtUtils/Command/MM.pm
+++ b/cpan/ExtUtils-MakeMaker/lib/ExtUtils/Command/MM.pm
@@ -146,7 +146,7 @@ sub pod2man {
 
         next if ((-e $man) &&
                  (mtime($man) > mtime($pod)) &&
-                 (mtime($man) > mtime("Makefile")));
+                 (mtime($man) > (mtime("Makefile") // 0)) );
 
         my $parser = Pod::Man->new(%options);
         $parser->parse_from_file($pod, $man)
